examples/arrays/arrays.go

```diff
@@ -41,4 +51,12 @@ func main() {
 		}
 	}
 	fmt.Println("2d: ", twoD)
+
+	// You can create and initialize multi-dimensional
+	// arrays at once too.
+	twoD = [2][3]int{
+		{1, 2, 3},
+		{1, 2, 3},
```

Comment:
I notice there's a mismatch now with the output - can you fix this?

Reply:
Done